conf/deployment/aws/upi_1az_rhcos_lso_3m_3w_disconnected.yaml

```diff
@@ -0,0 +1,18 @@
+---
+ENV_DATA:
+  platform: 'aws'
+  deployment_type: 'upi'
+  flexy_deployment: true
+  flexy_template: 'upi-on-aws/versioned-installer-disconnected'
+  skip_ntp_configuration: true
+  availability_zone_count: 1
+  master_replicas: 3
+  worker_replicas: 3
+  master_instance_type: 'm4.xlarge'
+  worker_instance_type: 'm5.4xlarge'
```

Comment:
as per doc we need to use i3 instance

Reply:
@prsurve the information what I get was, that i3 instances was mentioned in the time of OCS 4.4 as [Technology preview](https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.4/html-single/planning_your_deployment/index#infrastructure-requirements_rhocs). I didn't find any further information about this topic.

Reply:
@prsurve this is not meant to be use i3 but kind of hack to be able to use LSO deployment also on other instance types to have coverage for LSO disconnected mode. And with this approach when we have attached disks we are able to achieve this. (For sure it's not something recommended to users - they will use EBS for OSDs directly)